<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Course_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Course_Selection">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>271</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC 10B: Intermediate Programming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC 10C: Advanced Programming</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollBar" name="HW1_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW1_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW1_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HW2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 2</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW2_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW2_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW3_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW3_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 3</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW3_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW4_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW4_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 4</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW4_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW5_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW5_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 5</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW5_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW6_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW6_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 6</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW6_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW7_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW7_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 7</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW7_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="HW8_scroll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="HW8_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HW 8</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="HW8_spin">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>48</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Midterm1_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Midterm1_scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Midterm1_spin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Midterm2_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Midterm2_scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Midterm2_spin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Final_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Exam</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Final_scroll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="Final_spin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Scheme_A_Selection">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Scheme_B_Selection">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scheme B</string>
    </property>
   </widget>
   <widget class="QLabel" name="Final_Score_Display">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color:white}</string>
    </property>
    <property name="text">
     <string>         0</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
